doltpy/cli/tests/helpers.py

```diff
@@ -0,0 +1,27 @@
+import os
+from typing import Tuple
+from typing import List
+
+
+def get_repo_path_tmp_path(path: str, subpath: str = None) -> Tuple[str, str]:
```

Comment:
nit: i've noticed you like helper functions, but some of these take away from code readability for me

Reply:
I like them a lot, when you own the code I am happy to bring my contributions into line with your style :)